app/src/services/attestation.tsx

```diff
@@ -158,6 +210,87 @@ export const AttestationProvider: React.FC<AttestationProviderParams> = ({ child
     }
   }
 
+  const handleProofs = async (proof: ProofExchangeRecord, agent: BifoldAgent): Promise<void> => {
```

Comment:
Pretty long `fn` would be a good candidate to break into smaller more surgical `fn` and call them.